src/config.rs

```diff
@@ -35,10 +35,8 @@ use std::sync::Mutex;
 
 use crate::errors::*;
 
-lazy_static! {
-    static ref CACHED_CONFIG_PATH: PathBuf = CachedConfig::file_config_path();
-    static ref CACHED_CONFIG: Mutex<Option<CachedFileConfig>> = Mutex::new(None);
-}
+static CACHED_CONFIG_PATH: Lazy<PathBuf> = Lazy::new(CachedConfig::file_config_path);
+static CACHED_CONFIG: Lazy<Mutex<Option<CachedFileConfig>>> = Lazy::new(|| Mutex::new(None));
```

Comment:
`Mutex::new` is const since Rust 1.63 and https://github.com/mozilla/sccache/pull/1702 bumped sccache's MSRV to 1.64 three months ago so this can be simplified further I think 🙂